File: 2022/test/day_09_test.exs
defmodule Day09Test do
  use ExUnit.Case

  import TestHelper
  import Day09

  test "part1" do
    assert part1(input("day_09_example_1")) == 13
    assert part1(input("day_09")) == 6212
  end

  test "part2" do
    assert part2(input("day_09_example_1")) == 1
    assert part2(input("day_09_example_2")) == 36
    assert part2(input("day_09")) == 2522
  end
end

File: 2024/day09/day09_test.go
package day09

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestPart1(t *testing.T) {
	assert.Equal(t, 1928, Part1("input_example_1"))
	assert.Equal(t, 6390180901651, Part1("input"))
}

// func TestPart2(t *testing.T) {
// 	assert.Equal(t, 34, Part2("input_example_1"))
// 	assert.Equal(t, 1032, Part2("input"))
// }
